docs/questions/introduction.md

```diff
@@ -54,6 +54,12 @@ Questions that live in a dashboard are only visible in that dashboard. These que
 
 Questions saved to a collection can be added to multiple dashboards. Moving a question from one collection to another collection won't have any effect on the dashboards the question has been added to. To save a question to a collection, you'll need to be in a group with [curate access](../permissions/collections.md#curate-access) to that collection.
 
+### Checking for for breaking changes
+
+> {% include plans-blockquote.html feature="Checking for breaking changes" %}
+
+When a question, model, or metric is saved, Metabase will try to detect if the changes you made would break any other entities that depend on the entity you changed. For example, if you deleted or removed a column from a question that is referenced in other questions, Metabase will warn you that those downstream questions will break. 
```

Comment:
```suggestion
When a question, model, or metric is saved, Metabase will try to detect whether the changes you made would break any dependent entities. For example, if you removed a column from a question that other questions rely on, Metabase will warn you that those downstream questions will break. 
```